### PR TITLE
Milestone 3: Feature/add remove

### PR DESCRIPTION
**Pull request by alvp01**
# In this milestone:

In this milestone, I worked on the add, remove and edit functionality of the list by implementing the following:

- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for editing task descriptions.
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for the "Clear all completed" button.
- Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
- All changes to the To Do List should be saved in local storage.

### CODE REVIEW

**Comment by alvp01** [comment]
I added the gh-pages npm, and enabled the github pages before installing it. I don't know what's happening that it keeps showing the readme instead of the actual app. Any help would be appreciated.

**Comment by voscarmv** [comment]
> I added the gh-pages npm, and enabled the github pages before installing it. I don't know what's happening that it keeps showing the readme instead of the actual app. Any help would be appreciated.

Hi @alvp01 , I'll be your code reviewer today.

Check this StackOverflow question out to solve your issue. https://stackoverflow.com/questions/25320356/can-i-have-my-github-pages-index-html-in-a-subfolder-of-the-repository

I'll be providing you with a code review ASAP!

**Comment by alvp01** [comment]
> > I added the gh-pages npm, and enabled the github pages before installing it. I don't know what's happening that it keeps showing the readme instead of the actual app. Any help would be appreciated.
> 
> Hi @alvp01 , I'll be your code reviewer today.
> 
> Check this StackOverflow question out to solve your issue. https://stackoverflow.com/questions/25320356/can-i-have-my-github-pages-index-html-in-a-subfolder-of-the-repository
> 
> I'll be providing you with a code review ASAP!

Thanks! I'm going to try it ASAP.

File: src/index.js [hunk 1]
```diff
@@ -5,9 +5,16 @@ import {
...
+  toggleEdit, toggleOffEdit, editDescription, changeTrashIcon,
+} from './modules/addRemove.js';
+
 let tasksElements = document.querySelectorAll('.tasks-list li');
 let taskCheckboxes = document.querySelectorAll('.list-checkbox');
+const editInput = document.createElement('input');
+editInput.setAttribute('type', 'text');
+const clearBtn = document.querySelector('.btn-clear');
 
 let tasksArray = [
```
Comment (voscarmv) [inline src/index.js:19]: - [ ] Please remove all the hardcoded elements form the array. In this milestone, we're supposed to begin with an empty task array.

**Review by voscarmv — CHANGES_REQUESTED** [review]
Hi @alvp01 , 

Good job so far!
- I see you're not using a specifc `index` variable to keep track of the length of the array, in order to assign indexes to the new tasks being added. Instead, you use an `updateTaskElements` array every time there is a change in the array. Good work.

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- Please empty the initial array. In this milestone, we're supposed to not have any hardcoded tasks in the code.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Applying the changes from my suggested article from StackOverflow to fix your LiveDemo link.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (alvp01) [reply src/index.js]: I removed the hardcoded tasks. The solution you provided for the gh-pages didn't work unfortunately.

**Review by voscarmv — APPROVED** [review]
Hi @alvp01 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alvp01** [comment]
> Hi @alvp01 ,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> Congratulations! tada
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you for your time and work. Happy weekend.

**Pull request merged**